react/Wrapper.tsx

```diff
@@ -50,27 +46,41 @@ const Wrapper: FC<Props> = ({
     [assemblyOptions, product, selectedItem, selectedQuantity, selectedSeller]
   )
 
-  const isAvailable =
-    isEmptyContext || available != null
-      ? available
-      : Boolean(
-          selectedSeller &&
-            selectedSeller.commertialOffer &&
-            selectedSeller.commertialOffer.AvailableQuantity > 0
-        )
+  const checkAvailability = (availableProp: Props['available']) => {
+    if (isEmptyContext) {
+      return false
+    }
+    if (availableProp != null) {
+      return availableProp
+    }
+
+    return Boolean(
+      selectedSeller?.commertialOffer &&
+        selectedSeller.commertialOffer.AvailableQuantity > 0
```

Comment:
Wouldn't this work?

```suggestion
      selectedSeller?.commertialOffer.AvailableQuantity > 0
```

Reply:
Not like this :(
The `>` operator won't handle `undefined` values. Buuut, I could just check for `Boolean(selectedSeller?.commertialOffer.AvailableQuantity)`, right? That should cover values greater than 0. Also, it is safe to assume that `AvailableQuantity` is never negative, right? 😅

Reply:
I'd say that it is, but you'd never know 🤷‍♂